### PR TITLE
Removing duplicate codespace section in getting-started

### PR DESCRIPTION
**Pull request by jasonviviano**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Duplicate sections for codespace

### Summary
🗑️📝🧹

Removed the alert box about GitHub Codespaces from the getting-started guide. This change makes the documentation more streamlined and flexible for different user preferences.

> _`alert` box is gone_
> _simpler docs for Radius users_
> _Codespaces or not_

### Walkthrough
*  Removed alert box about GitHub Codespaces from `getting-started/index.md` ([link](https://github.com/radius-project/docs/pull/944/files?diff=unified&w=0#diff-16c6b1f038bfac9806df34f162b60508f10bfe6eb84a7b62f2a1d7c8d83fd23cL25-L30))

## Issue reference
#943

### CODE REVIEW

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Could you target 0.27 instead of edge please?

File: docs/config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseURL = "https://docs.radapp.io/"
+baseURL = "https://edge.docs.radapp.io/"
```
Comment (jasonviviano) [inline docs/config.toml]:
```suggestion
baseURL = "https://docs.radapp.io/"
```

File: docs/config.toml [hunk 2]
```diff
@@ -68,17 +68,17 @@ tag = "tags"
 [params]
 copyright = "Radius"
 
-version = "v0.27"
+version = "edge"
```
Comment (jasonviviano) [inline docs/config.toml]:
```suggestion
version = "v0.27"
```

File: docs/config.toml [hunk 3]
```diff
@@ -68,17 +68,17 @@ tag = "tags"
...
 tag_version = "latest"
-chart_version = "0.27.0"
+chart_version = "0.26.0"
```
Comment (jasonviviano) [inline docs/config.toml]:
```suggestion
chart_version = "0.27.0"
```

File: docs/config.toml [hunk 4]
```diff
@@ -68,17 +68,17 @@ tag = "tags"
...
 
 # Algolia Search
 algolia_docsearch = true
 
 # GitHub Information
 github_repo = "https://github.com/radius-project/docs"
 github_subdir = "docs"
-github_branch = "v0.27"
+github_branch = "edge"
```
Comment (jasonviviano) [inline docs/config.toml]:
```suggestion
github_branch = "v0.27"
```

File: docs/layouts/partials/hooks/body-end.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
     container: '#docsearch',
     appId: 'ED67NSVSZS',
     apiKey: '21e2920c3a9c29b98cbc382694413f29',
-    indexName: 'radapp-dev',
+    indexName: 'radapp-dev-edge',
```
Comment (jasonviviano) [inline docs/layouts/partials/hooks/body-end.html]:
```suggestion
    indexName: 'radapp-dev',
```

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**